src/utils/extensions/writeExtension.ts

```diff
@@ -163,3 +164,16 @@ export const isExtensionInitialised = async (
     mostRecentInitialisedLog.blockNumber >= installedLog.blockNumber
   );
 };
+
+export const removeExtensionInitialisedListeners = (
+  colonyAddress: string,
+): void => {
+  const eventProcessContext = getEventProcessorContext();
+  const removeListeners =
+    eventProcessContext.removeListeners[colonyAddress]?.[
+      RemoveListenerKeys.EXTENSION_INITIALISED
+    ];
```

Comment:
Uff. While I understand how this came about, it's really a mouth full to read :)